src/utils.py

```diff
@@ -20,15 +23,16 @@ def create_line_plot(df, x_column, y_columns, title, legend_names):
             title_font=dict(size=10),
             font=dict(size=8),
             x=1,  # Horizontally align to the right
-            y=0   # Vertically align to the bottom
+            y=0,  # Vertically align to the bottom
         ),
-        plot_bgcolor="white"
+        plot_bgcolor="white",
     )
 
     # Optionally remove the original column names from the hover data
     fig.update_traces(hovertemplate=None)
 
-    return fig
+    fig.show(config=config)
```

Comment:
This is what is making the line plots dissapear 

```suggestion
    return fig
```